.github/workflows/test.yml

```diff
@@ -0,0 +1,79 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  render:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+
+      - run: python -m pip install cookiecutter
+      - run: python -m cookiecutter . --no-input
+
+      - uses: actions/upload-artifact@v3
+        with:
+          name: rendered-project
+          path: wagtail-llama-save
+
+  lint:
+    needs: render
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: rendered-project
+          path: /tmp/wagtail-llama-save
+
+      - working-directory: /tmp/wagtail-llama-save
+        run: |
+          git init
+          git add .
+          python -m pip install pre-commit
+          pre-commit run --all-files --show-diff-on-failure --color=always
+
+  test:
+    needs: render
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: rendered-project
+          path: wagtail-llama-save
+
+      - working-directory: wagtail-llama-save
+        run: python -m pip install -e .[testing]
+      - working-directory: wagtail-llama-save
+        run: python testmanage.py test
```

Comment:
I haven't bothered with a build matrix or tox or anything here. Running `testmanage.py test` actually runs zero tests here, so I think all we are actually testing (if anything) is that the test runner can bootstrap. Tbh, I think we could consider dropping this job altogether.

Reply:
let's drop this since there are no tests

Reply:
OK. I've pushed 4c2cbe9 removing the test run. I think it probably is useful to still check that the package we generate is installable